app/question-component/question.css

```diff
@@ -0,0 +1,27 @@
+my-sample h1 {
+  text-align: center;
+  color: #FFF;
+  font-size: 72px;
+}
+
+.main-question-container {
```

Comment:
Always always always style your components with rules like 

```
quiz-question .main-question-container {

}
```

Never, never like 

```
.main-question-container {

}
```

The latter styles *every div in the app with that class*, including ones in other components. Isolate your styles to this component using a better selector.